### ISSUE TITLE
Doesn't work anymore with responsiveView: false

### ISSUE THREAD

**Issue by Wykks** [swiety85/angular2gridster#246]
Getting this error when I try to add a `gridster-item` with `responsiveView: false`:
```
DashletGridComponent.html:3 ERROR TypeError: Cannot read property 'direction' of undefined
    at GridsterItemComponent.setPositionsForGrid (gridster-item.component.js:303)
    at GridsterItemComponent.setPositionsOnItem (gridster-item.component.js:161)
    at GridsterItemComponent.ngOnInit (gridster-item.component.js:84)
```

Why does it need `options.breakpoint` to be set ?

(It was working fine in version 1.3.4)

**Comment by swiety85** [comment]
It's a bug. I will fix it as soon as possible. Thx for reporting.

**Comment by swiety85** [comment]
Fixed in v1.5.2
